<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>COM Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_comselect"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_comconnect">
          <property name="text">
           <string>Connect to Transfer Controller</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>System State:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_system_state">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Waiting to Cycle</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X-Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y-Position</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Z-Position</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="xaxis_counts">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>000000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yaxis_counts">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>000000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="zaxis_counts">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>000000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="xaxis_mm">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>000.00mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yaxis_mm">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>000.00mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="zaxis_mm">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>000.00mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Input/Output States</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_rtl_signal">
          <property name="text">
           <string>bReadyToLoadSRAS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_rts_signal">
          <property name="text">
           <string>bReadyToSRAS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_r3dsafe">
          <property name="text">
           <string>R3DSafetyOK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_srasready_signal">
          <property name="text">
           <string>bSRAS_Ready</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ctl_signal">
          <property name="text">
           <string>bClearToLoadSRAS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_srascomplete_signal">
          <property name="text">
           <string>bSRASComplete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_sraserror_signal">
          <property name="text">
           <string>bSRAS_ERROR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Position Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Robomet Handoff Position</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>x-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_r3dh_x"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>y-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_r3dh_y"/>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>z-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_r3dh_z"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_r3dh_sync">
          <property name="text">
           <string>Sync to Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_r3dh_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Z-Slide Transfer Position</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>x-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_stp_x"/>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>y-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_stp_y"/>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>z-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_stp_z"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_stp_sync">
          <property name="text">
           <string>Sync to Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_stp_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>SRAS Dropoff Position</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>x-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_sdp_x"/>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>y-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_sdp_y"/>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>z-pos [counts]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_sdp_z"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_sdp_sync">
          <property name="text">
           <string>Sync to Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_sdp_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_auto_toggle">
        <property name="text">
         <string>Switch to Auto Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
